response_operations_ui/__init__.py

```diff
@@ -14,6 +14,7 @@
 from response_operations_ui.logger_config import logger_initial_config
 from response_operations_ui.user import User
 from response_operations_ui.views import setup_blueprints
+from response_operations_ui.common.jinja_filters import blueprint as filter_blueprint
```

Comment:
Minor point, but couldn't you just name the blueprint in `common.jinja_filters` to `filter_blueprint` so you don't have to use `as` in the import statement?

Reply:
Done!